3.3/examples/django/cleanup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "-> removing openshift.local.* directories..."
+sudo rm -rf openshift.local.*
+
+echo "-> stopping all k8s docker containers..."
+docker ps | awk '{ print $NF " " $1 }' | grep ^k8s_ | awk '{print $2}' |  xargs -l -r docker stop
```

Comment:
Remove cleanup, as this is part of openshift stuff, you don't want to mess with this :wink: